index.html

```diff
@@ -74,173 +74,183 @@ <h2 class="works">My Recent<br> Works</h2>
           </ul>
-    </div>
-</div>
-
-<div class="griditem">
-  <div class=greyspace>
-    <div><img id="frameworks" src="./Assets/icon-frameworks.svg" alt="Icon"></div>
-    <h6>Frameworks </h6>
-        <ul class="minimenu">
+      </div>
+  
+    <div class="griditem2">
+      <div><img id="frameworks" src="./Assets/icon-frameworks.svg" alt="Icon"></div>
+      <h6>Frameworks </h6>
+        <ul class="minimenu2">
           <li class="button3">React.js</li>
           <li class="button3">Ruby on Rails</li>  
           <li class="button3">RSpec</li><br>      
-        </ul>
-        <ul class="minimenu">
           <li class="button3">Capybara</li> 
           <li class="button3">Selenium</li>       
         </ul>
-  </div>
-</div>
-
-<div class="griditem">
-  <div class=greyspace>
-    <div></div><img id="skills" src="./Assets/icon-skills.svg" alt="Skills"></div>
-    <h6>Skills</h6>
-        <ul class="minimenu">
+    </div>
+  
+    <div class="griditem2">
+    <div><img id="skills" src="./Assets/icon-skills.svg" alt="Skills"></div>
+      <h6>Skills</h6>
+        <ul class="minimenu2">
           <li class="button3">Database Management</li>
           <li class="button3">Version Control</li>   
-        </ul>
-
-        <ul class="minimenu"> 
           <li class="button3">CLI</li>
           <li class="button3">Web Developement</li>  
           <li class="button3">API Design</li>  
         </ul>
-  </div>
+    </div>
 </div>
-
 </section>
 
 </main>
 
-  <footer class="footer"></footer>
+<div class="wrapper">
+  <footer class="footer">
+  
+<div><p id=footertext>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p></div>
+  
+<div class="form">
+<form action="https://formspree.io/f/meqwblog" method="post">
+      <input type="text" id="name" name="user_name" placeholder="Full Name" maxlength="30">
+      <input type="email" id="mail" name="user_email" placeholder="Email Address" pattern="[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}" required>
+      <textarea id="msg" name="user_message" placeholder="Hey I'd like to connect to..." maxlength="500"></textarea>
+      <button type="submit" name="submit" class="button">Get in touch</button>
+</form>
+</div>
+
+<div class="email">victor@onsleer.co.za</div>
 
+<div class="social-footer2">
+  <ul>
+    <li><a href="https://twitter.com/dvcorreia1984"> <img src="./Assets/social/twitter.svg" alt="Twitter"></a></li>
+    <li><a href="https://www.linkedin.com/in/dvcorreia/"><img src="./Assets/social/Linkedin icon.svg" alt="Linkedin"></a></li>
+    <li><a href="#"><img src="./Assets/social/medium.svg" alt="Medium"></a></li>
+    <li><a href="https://github.com/dvcorreia1984"><img src="./Assets/social/github.svg" alt="Github"></a></li>
+    <li><a href="#"><img src="./Assets/social/angellist.svg" alt="Angellist"></a></li>
+  </ul>
+</div>
+
+<div id="bottomline"></div>
+
+  </footer>
+</div>
 
 </body>
 </html>
```

Comment:
[OPTIONAL] This is a pair programming activity, so please add the name of your partner to your Authors section in the README. Always remember that in pair programming activities both partners need to contribute equally to this project. The previous reviewer didn't mention this but I **highly** recommend you keep this in mind 👍 .

Reply:
Thank you for making me aware of this.